docs/src/modules/components/withRoot.js

```diff
@@ -9,7 +9,8 @@ import findPages from /* preval */ 'docs/src/modules/utils/findPages';
 import { loadCSS } from 'fg-loadcss/src/loadCSS';
 import acceptLanguage from 'accept-language';
 
-acceptLanguage.languages(['en', 'zh']);
+// acceptLanguage.languages(['en', 'zh']);
+acceptLanguage.languages(['en']);
```

Comment:
I guess this was by accident?

Reply:
It's not ready yet. @mbrookes is working on that.

Reply:
Odd to squeeze it into this PR though...